### PR TITLE
Added the search button hover style

### PR DESCRIPTION
**Pull request by Shivam-Shandilya1**
## What is the change?
Modified the search button style css by adding color to the hovering state.

## Related issue?
close: #576 

## How was it tested?
I have tested by running the localhost:3000 and inspecting through the google developer tool in my device.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.

https://user-images.githubusercontent.com/72553479/113506919-35b19c80-9565-11eb-8da0-87ca25d86958.mp4

### CODE REVIEW

**Comment by Shivam-Shandilya1** [comment]
Please review my pull request.

File: public/css/styles.css [hunk 1]
```diff
@@ -565,7 +565,16 @@ input[name='perPage'] {
...
+.search {
+    background: var(--color-primary);
+}
+.search:hover {
+    background: #e75d3b;
+}
+.search:active,
+.search:focus {
+    background: var(--color-secondary);
+}
```
Comment (Pranav016) [inline public/css/styles.css:577]: I think this same style exists for all the other buttons on home page, can you reuse that code? Make a single class for adding that style and add that to all the buttons..
I think that's the whole idea of using a class.
I know its an addition to the issue but I feel cleaner code would be easier to debug 👍

**Review by Pranav016 — CHANGES_REQUESTED** [review]

File: public/css/styles.css (hunk 1 above)
Comment (Shivam-Shandilya1) [reply public/css/styles.css:577]: Okay

**Comment by Pranav016** [comment]
comment again to reopen PR..

**Pull request closed without merging**